docs/installation.md

```diff
@@ -0,0 +1,130 @@
+
+# Installation Guide
+This document can be used as a quick start guide to setup the Dev Environment. Please review the system requirements listed below before moving forward with the SDO installation and deployment.
+## Table of Contents
```

Comment:
Table of contents, not rendering on new line each. Put "\<br>" at the end of each item to ensure that.